fastlane/Fastfile

```diff
@@ -139,12 +145,9 @@ lane :update_hybrid_common_versions do |options|
     UI.user_error!("Missing `version` argument")
   end
 
-  previous_version_number = get_phc_version
-
-  # Add to options so if we're chaining lanes together, the other lanes get it
-  options[:previous_hybrid_version] = previous_version_number
```

Comment:
Just to check, the `options[:previous_hybrid_version]` is not used?

Reply:
No. It also doesn't really work like this. If you add something to `options` in a lane, it's not accessible for the following lanes that are being executed. This basically didn't work anyway